wiki/Tournaments/CFC/CFIC_3/fr.md

```diff
@@ -0,0 +1,358 @@
+---
+tags:
+  - CFIC
+  - CFIC3
+---
+
+# Catch French Intermediate Cup 3
+
+![CFIC 3 logo](img/logo.jpg)
+
+L'**Catch French Intermediate Cup 3** (***CFIC 3***) était un tournoi français intermédiaire à double élimination 1v1 osu!catch organisé par ::{ flag=FR }:: [Hecatia15](https://osu.ppy.sh/users/3163012) et ::{ flag=FR }:: [YANOO123](https://osu.ppy.sh/users/5122949). Les joueurs français classés entre 900 et 2 999 ont pu participer. C'était la troisième édition de l'Catch French Intermediate Cup et fait partie de l'Catch French Cup.
+
+## Dates du tournoi
+
+| Évènement | Dates |
+| --: | :-- |
+| Phase d'inscription | 2021-08-09/2021-08-29 |
+| Phase de screening | 2021-08-30/2021-09-10 |
+| Live du tirage au sort | 2021-09-05 (18:00 UTC+2) |
+| Qualifications | 2021-09-11/2021-09-12 |
+| Huitièmes de finale | 2021-09-18/2021-09-19 |
+| Quarts de finale | 2021-09-25/2021-09-26 |
+| Demi-finale | 2021-10-02/2021-10-03 |
+| Finale | 2021-10-09/2021-10-10 |
+| Grande Finale | 2021-10-17 |
+
+## Prix
+
+| Placing | Prix |
+| :-: | :-- |
+| ![Couronne d'or](/wiki/shared/crown-gold.png "1ère place") | Badge de profil unique, 8 mois d'osu!supporter |
+| ![Couronne d'argent](/wiki/shared/crown-silver.png "2ème place") | 6 mois d'osu!supporter |
+| ![Couronne de bronze](/wiki/shared/crown-bronze.png "3ème place") | 4 mois d'osu!supporter |
+| *4ème place* | 2 mois d'osu!supporter |
+
+![](img/badge.jpg "Badge des gagnants de l'CFC3")
+
+## Organisation
+
+L'Catch French Intermediate Cup 3 était organisé par différents membres.
+
+| Position | Membre(s) |
```

Comment:
Not sure about "Position".

Reply:
I guess we'll leave it as it is for now, can always improve later on.